demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -28,6 +28,9 @@
     get_files_in_dir, get_pack_name, is_v2_file, print_error,
     server_version_compare)
 
+DEPRECATED_DESC_REGEX = r"Deprecated\.\s*(.*?Use .*? instead\.*?)"
```

Comment:
probably better to put these in `constats.py` so you wont have to init them both here and in scripts